dist/src/main/config/broker.yaml.template

```diff
@@ -313,6 +313,7 @@
           # connectionString:
 
           # Used to define the container name where the blobs will be saved.
+          # This value can be left empty, but default word 'null' will be used.
```

Comment:
:bulb: Ahh, "null" is used, that's why it works :smile: 

:x: IMO this looks weird and we need to make a choice if we want to require a container name or if we want to support _no_ container name explicitly.

Reply:
Right 🤔  it didn't look great to have a container with the name "null/...." . 
I removed the option to have it null by validating the container name as well. Let me know if you agree.